projects/storefrontapp/src/environments/environment.ts

```diff
@@ -28,12 +28,16 @@ export const environment: Environment = {
   cdc: buildProcess.env.CX_CDC ?? false,
   cdp: buildProcess.env.CX_CDP ?? false,
   cpq: buildProcess.env.CX_CPQ ?? false,
+
   digitalPayments: buildProcess.env.CX_DIGITAL_PAYMENTS ?? false,
   epdVisualization: buildProcess.env.CX_EPD_VISUALIZATION ?? false,
   s4om: buildProcess.env.CX_S4OM ?? false,
   segmentRefs: buildProcess.env.CX_SEGMENT_REFS ?? false,
   opps: buildProcess.env.CX_OPPS ?? false,
   requestedDeliveryDate: buildProcess.env.CX_REQUESTED_DELIVERY_DATE ?? false,
+  estimatedDeliveryDate: buildProcess.env.CX_ESTIMATED_DELIVERY_DATE ?? false,
```

Comment:
MAJOR: why are the estimated delivery change codes present as part of PR? can you please remove them?

Reply:
After pulling the Develop branch, this is what's coming up.